src/Support/Arr.php

```diff
@@ -49,6 +49,24 @@ public static function arrayMergeRecursiveDistinct(array &$array1, array &$array
         return $merged;
     }
 
+    /**
+     * @param $array1
+     * @param $array2
+     *
+     * @return array
+     */
+    public static function mergeAfterEachOther($array1, $array2): array
```

Comment:
typing?

Reply:
$array1 is een object (object type kan pas vanaf PHP 7.2). Momenteel supporten wij ook nog PHP 7.1.
Bij array2 heb ik het aangegeven